<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>226</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Class E</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>209</width>
        <height>416</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
       <item row="1" column="1">
        <widget class="QLineEdit" name="leVcc"/>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLabel" name="label_5">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="text">
          <string>Ω</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="leC2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="powSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Output power</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;P&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leQuality"/>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="freqUnit">
         <property name="text">
          <string>MHz</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>C&lt;sub&gt;2&lt;/sub&gt;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="leL1"/>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="qualityUnit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="vccSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Supply voltage</string>
         </property>
         <property name="text">
          <string>V&lt;sub&gt;CC&lt;sub&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>pF</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>L&lt;sub&gt;2&lt;/sub&gt;</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2" colspan="2">
        <widget class="QLabel" name="l1Unit">
         <property name="text">
          <string>μH</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="freqSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Frequency</string>
         </property>
         <property name="text">
          <string>f</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>pF</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="leRes"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="leVo"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLabel" name="powUnit">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLabel" name="voUnit">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="4">
        <widget class="QPushButton" name="calc">
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="qualitySymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Quality factor</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Q&lt;span style=&quot; vertical-align:sub;&quot;&gt;L&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leFreq"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="l1Symbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>min. inductivity of L&lt;sub&gt;1&lt;/sub&gt;</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;L&lt;span style=&quot; vertical-align:sub;&quot;&gt;1(min)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lePow"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="vccUnit">
         <property name="text">
          <string>V</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="voSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Transistor saturation offset voltage</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;V&lt;span style=&quot; vertical-align:sub;&quot;&gt;O&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>C&lt;sub&gt;1&lt;/sub&gt;</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLineEdit" name="leL2"/>
       </item>
       <item row="11" column="2" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>μH</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="leC1"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Primary</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>209</width>
        <height>341</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
       <item row="3" column="0">
        <widget class="QLabel" name="widthSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;effective width (r&lt;span style=&quot; vertical-align:sub;&quot;&gt;outter&lt;/span&gt; - r&lt;span style=&quot; vertical-align:sub;&quot;&gt;inner&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>W</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="leL1_p"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="angleSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>rise angle</string>
         </property>
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="nUnit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2">
        <widget class="QLabel" name="widthUnit">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="leHeight"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="nSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>number of turns</string>
         </property>
         <property name="text">
          <string>N</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="leWidth"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;L&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="leAngle"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="l1Symbol_2">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>min. inductivity of L&lt;sub&gt;1&lt;/sub&gt;</string>
         </property>
         <property name="text">
          <string>L</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="leL_p"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;L&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="radiusSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>average radius</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;R&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="heightSymbol">
         <property name="cursor">
          <cursorShape>WhatsThisCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>effective height</string>
         </property>
         <property name="text">
          <string>H</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leN"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="QLabel" name="voUnit_2">
         <property name="text">
          <string>º</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>μH</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>μH</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2" colspan="2">
        <widget class="QLabel" name="l1Unit_2">
         <property name="text">
          <string>μH</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLabel" name="heightUnit">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QPushButton" name="calc_2">
         <property name="text">
          <string>Calculate</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leRadius"/>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="leL2_p"/>
       </item>
       <item row="1" column="2" colspan="2">
        <widget class="QLabel" name="radiusUnit">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>226</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>calc</tabstop>
  <tabstop>leFreq</tabstop>
  <tabstop>leVcc</tabstop>
  <tabstop>lePow</tabstop>
  <tabstop>leQuality</tabstop>
  <tabstop>leVo</tabstop>
  <tabstop>leRes</tabstop>
  <tabstop>leC1</tabstop>
  <tabstop>leC2</tabstop>
  <tabstop>leL2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
